GITFLOW.md

```diff
@@ -113,7 +111,7 @@ No need to worry about modifying the `API Documentation`,  everything is generat
 ## Hotfixes
 
 A hotfix is a patch that needs to go directly into `master` without going through the regular release process.
-The most common use case is to patch a bug that's on production when `staging` contains code that isn't yet ready for release.
+The most common use case is to patch a bug that's on production when `hotfix` contains code that isn't yet ready for release.
 
```

Comment:
Another use case is when a past release needs a patch. For exemple we are currently on version 3.2 but find a critical bug that is present since 2.5 and want to fix it. Then we would create a hotfix branch and release it as 2.5.1